src/main/java/com/blakebr0/extendedcrafting/item/ItemSingularity.java

```diff
@@ -37,8 +36,12 @@ public ItemSingularity() {
 
 	@Override
 	public String getItemStackDisplayName(ItemStack stack) {
-		String name = items.containsKey(stack.getMetadata()) ? items.get(stack.getMetadata()).getName().replace("_", " ") : "Dummy";
-		return WordUtils.capitalize(name) + " " + Utils.localize("item.ec.singularity.name");
+		String localizedMaterialName = "Invalid";
+		if (items.containsKey(stack.getMetadata())) {
+			String materialName = items.get(stack.getMetadata()).getName();
+			localizedMaterialName = I18n.translateToLocal("item.ec.singularity." + materialName);
+		}
+		return I18n.translateToLocalFormatted("item.ec.singularity.name", localizedMaterialName);
```

Comment:
Why can't you just return `I18n.translateToLocal("item.ec.singularity." + materialName)`? I don't see any formatting being applied that would necessitate using `translateToLocalFormatted`.

Reply:
The material name translation is two parts.

The first part would find the key for example `item.ec.singularity.aluminum`, which would return the localized string `Aluminum`.

The second one translates the key `item.ec.singularity.name` which resolves to `%s Singularity` and applies the localized material name with a formatting variable, resulting in the string `Aluminum Singularity`.